src/dmd/chkformat.d

```diff
@@ -493,7 +518,7 @@ private:
  * Returns:
  *      Format
  */
-pure nothrow @safe
+nothrow @safe
```

Comment:
You could keep the attribute by passing `bool useGNUExts = findCondition(...)` as a default argument.

Reply:
Nice trick !